### ISSUE TITLE
It broke after installing a few things

### ISSUE THREAD

**Issue by Ridianoid** [Magisk-Modules-Alt-Repo/HideNavBar#92]
Greetings. Due to the fact that the banks in my country have improved and activated the rum and root detection systems, I did something by following a tutorial and this tweak that I have been using for years, which worked perfectly, is broken and now it is not working. What caused this and how can I fix it? 

I followed this tutorial; https://telegra.ph/Mona-11-29-3

Installed;
-Magisk Alpha
-Zygisk next
-Shamiko
-Play Integrity Fix
-Tricky Store 
-Hide My Root
-Lsposed Irena 
-Hide My Applist

**Comment by DanGLES3** [comment]
> Greetings. Due to the fact that the banks in my country have improved and activated the custom rom and root detection systems, I did something by following a tutorial and this tweak that I have been using for years, which worked perfectly, is broken and now it is not working. What caused this and how can I fix it?
> 
> I followed this tutorial; https://telegra.ph/Mona-11-29-3
> 
> Installed; -Magisk Alpha -Zygisk next -Shamiko -Play Integrity Fix -Tricky Store -Hide My Root -Lsposed Irena -Hide My Applist

Is your Shamiko set to Whitelist mode perhaps?

**Comment by Ridianoid** [comment]
> > Greetings. Due to the fact that the banks in my country have improved and activated the custom rom and root detection systems, I did something by following a tutorial and this tweak that I have been using for years, which worked perfectly, is broken and now it is not working. What caused this and how can I fix it?
> > I followed this tutorial; https://telegra.ph/Mona-11-29-3
> > Installed; -Magisk Alpha -Zygisk next -Shamiko -Play Integrity Fix -Tricky Store -Hide My Root -Lsposed Irena -Hide My Applist
> 
> Is your Shamiko set to Whitelist mode perhaps?

Yes, magisk said something like that but it was impossible for me to use the phone like this, I decided to uninstall each module and magisk and reboot one by one. I found that many modules were not needed for me to use the application. I solved my problem with zygisk-next and playintegrity fix. I also installed navtweak and it works fine. I am amazed at the mind that made this ridiculous bottom space on stock android.
